app/components/groups/add-groups.jade

```diff
@@ -0,0 +1,81 @@
+div.row.new-groups-inp(
+  ng-controller="AddGroupController"
+  ng-init="actions.action='default'"
+  ng-switch="actions.action"
+)
+  div.col-md-6.col-md-offset-3.new-group(
+    ng-switch-when="default"
+    ng-click="actions.action='new-task'"
+  )
+    span New group
+    span.ctl-pane.pull-right
+      i.glyphicon.glyphicon-th-list.ctrl-btn(
+        tooltip-placement="bottom"
+        tooltip="Create new list"
+        ng-click="$event.stopPropagation()"
+        )
+      i.glyphicon.glyphicon-search.ctrl-btn(
+        tooltip-placement="bottom"
+        tooltip="Search"
+        ng-click="actions.action='search'"
+        )
+  div.col-md-6.col-md-offset-3.new-group(
+    ng-switch-when="new-task"
+    )
+    div.task-title(
+      contentEditable="true"
+      ng-model="newGroup.title"
+      placeholder="Input a title"
+      ng-init="view.isList=false"
+    )
+    div.task-description(
+      contentEditable="true"
+      ng-model="newGroup.tasks[0].description"
+      placeholder="Description" ng-if="!view.isList"
+      )
+    div.new-tasks(
+      ng-if="view.isList"
```

Comment:
one attribute can be written in one line like `.new-tasks(ng-if="view.isList")`. And don't use explicit `div`